astropy/coordinates/solar_system.py

```diff
@@ -137,7 +135,7 @@ def _get_earth_body_vector(time, body, earth_time=None):
         The solar system body to calculate.
 
         The allowed values for ``body`` can be found in
-        `~astropy.coordinates.SOLAR_SYSTEM_BODIES`.
+        ``astropy.coordinates.SOLAR_SYSTEM_BODIES``.
```

Comment:
Maybe this didn't get picked up by sphinx because there was no docstring below `SOLAR_SYSTEM_BODIES`? Anyway, not a big deal.